exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
we have our own styling for perseus, I don't see there any other ways around this.